README.md

```diff
@@ -32,7 +32,7 @@ more information on using dbt with BigQuery, consult [the docs](https://docs.get
 ## Reporting bugs and contributing code
 
 - Want to report a bug or request a feature? Let us know on [Slack](http://community.getdbt.com/), or open [an issue](https://github.com/dbt-labs/dbt-bigquery/issues/new)
-- Want to help us build dbt? Check out the [Contributing Guide](https://github.com/dbt-labs/dbt/blob/HEAD/CONTRIBUTING.md)
+- Want to help us build dbt? Check out the [Contributing Guide](https://github.com/dbt-labs/dbt-bigquery/blob/main/CONTRIBUTING.md)
```

Comment:
I noticed this the other day too! Thanks for the fix, any reason to hardcode main and not use `HEAD`?

Reply:
I copy pasted the link from my browser when I was browsing from Github UI so since main is the default right now it's how it is. I guess we could use both, maybe `HEAD` is a bit more durable because it's not related to the actual branch.

Reply:
It's small but might as well:
```suggestion
- Want to help us build dbt? Check out the [Contributing Guide](https://github.com/dbt-labs/dbt-bigquery/blob/HEAD/CONTRIBUTING.md)
```

Reply:
Ok, I've added your change 👍